source/reference/api/commit.txt

```diff
@@ -16,6 +16,8 @@ Description
 -----------
 
 Commits the synchronization operation to the destination cluster.
```

Comment:
[suggestion] It looks like the first half of the new sentence is saying virtually the same thing as the first sentence that's already here. Maybe we can just merge these two?
```suggestion
Finalizes the synchronization operation between the source and destination clusters. ``commit``stops continuous...
```

Reply:
My only hesitation here is that the the other endpoints use the endpoint name to describe the definition of the endpoint behavior. I do like the idea of combining the sentences and using "finalize" rather than "commit" but just wanted to check in re consistency

Reply:
Gotcha, that makes sense! I think the main concern is that the original JIRA ticket mentions that the "commit" language is what was causing confusion with the definition in the first place, so I think that could warrant overriding consistency in favor of clarity.

Reply:
Sounds good, just made the change!